plugins/node/opentelemetry-instrumentation-pino/src/types.ts

```diff
@@ -25,4 +24,4 @@ export interface PinoInstrumentationConfig extends InstrumentationConfig {
   logHook?: LogHookFunction;
 }
 
-export type Pino = typeof pino;
+export type Pino = any;
```

Comment:
This doesn't seem a good aproach. This leaks into the public exported class `PinoInstrumentation` therefore changing this back into a more concrete type could break user code.

Reply:
@Rauno56 Do you want to do something about it or should I revert your PR?

Reply:
Yeah, agreed - I thought I changed all the added `any`s to proper types in later commits, but missed this or left some commits unpushed. My bad. The changes from @mentos1386 did it the correct way regarding this so all good!